ambry-server/src/integration-test/java/com.github.ambry.server/StatsManagerTest.java

```diff
@@ -65,22 +79,27 @@
    */
   @After
   public void cleanup() throws IOException {
-    assertTrue(tempDir.getAbsolutePath() + " could not be deleted", StoreTestUtils.cleanDirectory(tempDir, true));
+    File[] files = tempDir.listFiles();
```

Comment:
can you move this file to "test" directory. As of now it is in "integration-test".